build/args/ffmpeg.gn

```diff
@@ -1,3 +1,6 @@
-import("release.gn")
+import("all.gn")
+is_component_build = false
+is_component_ffmpeg = true
+is_official_build = true
```

Comment:
What prompted this change -- ie, all three of these settings are the same in `release.gn`?

Is this change to get away from the `is_cfi = false` line in `release.gn`?

Reply:
Yup thats one of the reason, also since this is a separate target I would prefer it overrides from the common rather than release, it makes it easier to reason about the overrides.